cfme/tests/automate/custom_button/test_custom_menu.py

```diff
@@ -54,4 +79,11 @@ def test_custom_menu_display():
     Bugzilla:
         1678151
     """
-    pass
+    appliance.update_advanced_settings(ADVANCE_SETTING_CUSTOM_MENU)
+    appliance.reboot()
```

Comment:
**Question** Do we need a reboot here, or can we just do a evmserverd restart?

Reply:
@digitronik I just tested this and an evmserverd restart should work for this test, it adds the menus after the webui starts back up

Reply:
Going to mark this **Required**, I don't think we should be rebooting the appliance.